test/StateL.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+var FL = require('fantasy-land');
+
+
+var StateL = function StateL(f) {
```

Comment:
Hmm, not to confuse it with ```State``` monad, it just applicative that accumulate it results from ```left```, so ```L``` there stand for ```Left```